### PR TITLE
add italian language

### PR DESCRIPTION
**Pull request by piojpub**
Hi!
May you please add italian language to your project?

### CODE REVIEW

**Comment by blockbasti** [comment]
Hey, thanks for the contribution!
Could you also copy the /fastlane/metadata/android/de-DE folder and translate the 2 files in there into italian, so the F-Droid page is also translated?

**Comment by piojpub** [comment]
Yep, I will do it!
I'm really glad to contribute, that's my first time on github!

File: lib/l10n/intl_it.arb [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+  "licenses": "Licenza",
+  "viewLicense": "Vedi la licenza",
+  "viewOSSLicenses": "Vedi la licenza Open Source",
+  "ossLicenses": "Licenza Open Source",
+
+  "settings": "Impostazioni",
+  "language": "Lingua",
+  "english": "Inglese",
+  "german": "Tedesco",
+  "italian": "Italiano
```
Comment (blockbasti) [inline lib/l10n/intl_it.arb:26]: You're missing a closing quotation mark (") here

File: lib/l10n/intl_it.arb [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+  "deleteSet": "Cancella l'impostazione",
+  "repetitions": "Ripetizioni:",
+  "addExercise": "Aggiungi un esercizio",
+  "addRest": "Aggiungi una pausa",
+  "moveSetUp": "Muovi la sessione verso l'alto",
+  "moveSetDown": "Muovi la sessione verso il basso",
+  "deleteExercise": "Cancella l'esercizio",
+  "moveExerciseUp": "Muovi l'esercizio verso l'alto",
+  "moveExerciseDown": "Muovi la sessione verso il basso",
+  "name": "Nome
```
Comment (blockbasti) [inline lib/l10n/intl_it.arb:55]: Another missing closing "

**Comment by piojpub** [comment]
ok, sorry!
I will double check the files.

**Comment by blockbasti** [comment]
I've made some additional changes and merged your changes manually. Thank you for the contribution. Im closing this pull request now.

One more thing. If in the future more strings are added to the app, would you be able to translate them aswell? I don't speak italian and don't want to use google translate as that always produces some weird results.

**Comment by piojpub** [comment]
Hi!
Yes sure, I will very glad!

**Pull request closed without merging**